src/components/ActionCard.tsx

```diff
@@ -92,23 +52,44 @@ const ActionCard: React.FC<IProps> = ({
   const isImageURL = typeof image === "string"
 
   return (
-    <Card to={to} className={className} hideArrow={true}>
-      <ImageWrapper
-        isRight={isRight}
-        isBottom={isBottom}
+    <Link
+      boxShadow="
+	  0px 14px 66px rgba(0, 0, 0, 0.07),
+    0px 10px 17px rgba(0, 0, 0, 0.03), 0px 4px 7px rgba(0, 0, 0, 0.05)"
+      color="text"
+      flex="1 1 372px"
+      _hover={LinkFocusStyles}
+      _focus={LinkFocusStyles}
+      to={to}
+      className={className}
+      hideArrow={true}
+      m={4}
+      textDecoration="none"
+    >
+      <Flex
+        minH={"260px"}
+        bg={"cardGradient"}
+        direction={"row"}
+        justify={isRight ? "flex-end" : "center"}
+        align={isBottom ? "flex-end" : "center"}
         className="action-card-image-wrapper"
+        boxShadow="inset 0px -1px 0px rgba(0, 0, 0, 0.1)"
       >
         {!isImageURL && <Image image={image} alt={alt || ""} />}
```

Comment:
Lets use the `Image` from Chakra. https://chakra-ui.com/docs/components/image/usage

```tsx
<Image as="GatsbyImage" ...>
```